src/beacons/use-beacons.ts

```diff
@@ -0,0 +1,191 @@
+import { useIsBeaconsEnabled } from './use-is-beacons-enabled';
+import { KETTLE_API_KEY } from '@env';
+import { useCallback, useEffect, useState } from 'react';
+import { NativeModules, Platform } from 'react-native';
+import { Kettle, KettleModules } from 'react-native-kettle-module';
+import { KettleConsents } from 'react-native-kettle-module';
+import { PERMISSIONS, Permission, RESULTS, check, checkMultiple, request } from 'react-native-permissions';
+
+const BEACONS_CONSENTS = [KettleConsents.SURVEYS, KettleConsents.ANALYTICS];
```

Comment:
Should this be retrieved from remote config?

Reply:
Good question, since this is the only thing we need from them, not sure if in the future we should change that. But in the case is in the remote config we can not just change to new consents because haven't been approved.

Reply:
Good point, it would probably require more logic which is perhaps unnecessary now